core/cmd/shell.go

```diff
@@ -145,62 +137,52 @@ func (n ChainlinkAppFactory) NewApplication(ctx context.Context, cfg chainlink.G
 
 	dbListener := cfg.Database().Listener()
 	eventBroadcaster := pg.NewEventBroadcaster(cfg.Database().URL(), dbListener.MinReconnectInterval(), dbListener.MaxReconnectDuration(), appLggr, cfg.AppID())
-	ccOpts := evm.ChainSetOpts{
-		Config:           cfg,
-		Logger:           appLggr,
-		DB:               db,
-		KeyStore:         keyStore.Eth(),
-		EventBroadcaster: eventBroadcaster,
-		MailMon:          mailMon,
-	}
-
 	loopRegistry := plugins.NewLoopRegistry(appLggr.Named("LoopRegistry"))
 
-	var chains chainlink.Chains
-	chains.EVM, err = evm.NewTOMLChainSet(ctx, ccOpts)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to load EVM chainset")
+	// create the relayer-chain interoperators from application configuration
+	relayerFactory := chainlink.RelayerFactory{
+		Logger:       appLggr,
+		DB:           db,
+		QConfig:      cfg.Database(),
```

Comment:
Does this stands for QueryConfig? 

Are there other DBs associated with relayerFactories that would prevent this from being named `DbConfig` - or is it important that the creater of a RelayerFactory know that the DB is scoped to queries and not writes?